### PR TITLE
Serial scanning

### PR DESCRIPTION
**Pull request by ldbonkowski**
### Describe your changes

This PR accomplishes the following:

- Adds scanner selector dialog
- Scans serial number from image and loads it into the edit text field

### Issue ticket number and link

This PR addresses: #11 

### Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Necessary java docs are present
- [ ] Necessary UML diagrams are created
- [ ] Necessary tests are written

### Screenshots (if applicable)

e.g. UI screenshots, UML screenshots, etc

### CODE REVIEW

File: app/src/main/java/com/example/househomey/scanner/ImageScanner.java
Comment (owencooke) [inline app/src/main/java/com/example/househomey/scanner/ImageScanner.java:1]: I'm assuming this is meant to be inherited by Sami later on?

**Review by owencooke — APPROVED** [review]
Looks really good!

File: app/src/main/java/com/example/househomey/scanner/ImageScanner.java
Comment (Sami-Jagirdar) [reply app/src/main/java/com/example/househomey/scanner/ImageScanner.java:1]: Yep, both serial scanner and barcode scanner inherit from this class. Lukas did a sick job making this really easy to work with

**Review by MatthewNeufeld — APPROVED** [review]
Super clean. LGTM.

**Pull request merged**